k8sinterface/k8sconfig.go

```diff
@@ -186,10 +186,9 @@ func SetConfigClusterServerName(contextName string) {
 	ConfigClusterServerName = contextName
 }
 
-func GetK8sConfigClusterServerName() string {
+func GetK8sConfigClusterServerName(config *clientcmdapi.Config) string {
 	if ConfigClusterServerName == "" {
-		config := GetConfig()
-		if _, exist := config.Clusters[config.CurrentContext]; !exist {
+		if _, exist := config.Clusters[config.CurrentContext]; exist {
```

Comment:
add a test that if tomorrow someone changes `exist` to `!exist` it will fail, since this was the original bug